### PR TITLE
Comment on UpDownType usage in Rollershutter item (#3297)

### PR DESCRIPTION
**Pull request by n-kristev**
Signed off by: Nikolay Kristev (n.kristev@prosyst.com)

### CODE REVIEW

File: bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/items/RollershutterItem.java [hunk 1]
```diff
@@ -37,6 +37,11 @@
 
     static {
         acceptedDataTypes.add(PercentType.class);
+		/*
```
Comment (sjka) [inline bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/items/RollershutterItem.java]: Honestly, I'm not really sure whether this is the right place for such an explanation. It's not even shown in the JavaDoc for RollershutterItem. Somebody would need to look inside the sources and scroll to the right position in order to find it. 

IMHO the [items documentation](https://github.com/eclipse/smarthome/blob/master/docs/documentation/concepts/items.md) would be the much better place for such explanations.

**Comment by n-kristev** [comment]
Done, moved the comment to items.md

**Comment by maggu2810** [comment]
@sjka I assume your change request is now fulfilled.

**Comment by maggu2810** [comment]
@n-kristev There is a missing ip-validation. Can you squash your commits?

**Comment by kaikreuzer** [comment]
@maggu2810 I am not really happy with that addition - it adds some very specific details on RollershutterItem on an overview page where all item types are only briefly described. That imho brings some bad proportions to the page. 

We should imho better document in general how commands vs. state updates in items work and how the different item types work - I wouldn't treat Rollershutter as some prominent exception here.

**Comment by n-kristev** [comment]
@kaikreuzer OK, in this case I can change the note to something more general (if you want to keep it on the same page), and add Rollershutter and Dimmer as an example - in Dimmer's case, it is ON/OFF and the level which have a similar relation as the shutter UP/DOWN and level.

**Comment by n-kristev** [comment]
@maggu2810 Done, commits are squished and the IP validation passes now.

**Comment by n-kristev** [comment]
@kaikreuzer @sjka I have updated the note to be more generic re. item types.

**Comment by sjka** [comment]
Fully makes sense now, I like how this turned out, thanks!

**Comment by kaikreuzer** [comment]
@n-kristev Many thanks and sorry for my sluggish responsiveness...

**Pull request merged**